tests/time_tests/src/timetests/timetest_infer.cpp

```diff
@@ -60,21 +48,19 @@ int runPipeline(const std::string &model, const std::string &device, const bool
               cnnNetwork = ie.ReadNetwork(model);
               batchSize = cnnNetwork.getBatchSize();
             }
-
             {
               SCOPED_TIMER(load_network);
-              exeNetwork = ie.LoadNetwork(cnnNetwork, device, vpuConfig);
+              exeNetwork = ie.LoadNetwork(cnnNetwork, device);
             }
           }
         }
         else {
-          SCOPED_TIMER(load_network);
+          SCOPED_TIMER(load_network_cache);
```

Comment:
Two questions:
* Why do we need to distinguish LoadNetwork for cache/non-cache? I am just wondering
* I do not really understand how models are put into the cache? The cache should be empty at first run of the tool, according to https://docs.openvino.ai/latest/openvino_docs_IE_DG_Model_caching_overview.html

Reply:
The same name of the metric for the cache and without can confuse

Working with the model cache directory is regulated in the runner scripts (we use pytest). There is no work with the cache directory inside tool at the moment